src/components/shared/AmountItem/AmountItem.js

```diff
@@ -25,15 +25,24 @@ class AmountItem extends Component {
     title: PropTypes.string,
   };
 
-  renderChangeIcon() {
+  renderChange() {
```

Comment:
`renderTrend()` would be more accurate, since the first rendered node here is `<Trend>`, IMO